controllers/connector_controller.go

```diff
@@ -188,16 +190,66 @@ func (r *ConnectorReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Complete(r)
 }
 
+// returns a Connector StatefulSet object
+func (r *ConnectorReconciler) generateStsForConnector(connector *vanusv1alpha1.Connector) *appsv1.StatefulSet {
```

Comment:
suggest using full name, rename to generateStatefulSetForConnector

Reply:
modify